opba-db/src/main/resources/migration/migrations/0000-create-table-banks.xml

```diff
@@ -37,6 +37,7 @@
                 <constraints unique="true"/>
             </column>
             <column name="preferred_approach" type="VARCHAR(32)"/>
+            <column name="force_preferred_approach" type="boolean" defaultValue="false"/>
```

Comment:
Given the PSD2 logic 'force_preferred_approach' name looks misleading. We can't force EMBEDDED mode unfortunately. 'try_to_use_preferred_approach' seem a better name

Reply:
Also, doesn't seem that this is nullable column

Reply:
done